agent/lib/memory-review/memory-review-unbound-recovery.integration.test.ts

```
@@ -0,0 +1,203 @@
+        expect(locks.rows[0]!.waiting).toBe(true);
+      }, { timeout: 2_000, interval: 20 });
+      // Alert insertion takes KEY SHARE on the batch for its FK. It must not wait for the
+      // recovery transaction, which already waits for this transaction's lane lock.
+      await expect(readMemoryReviewLaneHealth(monitoring)).resolves.toHaveLength(1);
+    } finally {
+      await monitoring.query("ROLLBACK");
+      const recoveryError = await recovery;
+      await recovering.query("ROLLBACK");
+      monitoring.release();
+      recovering.release();
+      if (recoveryError) throw recoveryError;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)